Source/RealmResultsCache.swift

```diff
@@ -59,12 +59,17 @@ class RealmResultsCache<T: Object> {
 
     //MARK: Population
     func populateSections(objects: [T]) {
-        for object in objects {
-            let section = getOrCreateSection(object)
-            section.insertSorted(object)
+        let sections: [Section<T>] = objects.map {
+            let section = getOrCreateSection($0)
+            section.insert($0)
+            return section
+        }
+        
+        Set(sections).forEach {
```

Comment:
Do we really need to create a Set? Get or create section manages the section in order to not have dupplicates and separate objects by `sectionKeyPath`

One liner `.forEach { $0.sort() }` please :pray: